### DISCUSSION TITLE
(bug) Using the Windows 10 Start Menu with rounded corners, opening the hamburger menu makes the left corners square instead of rounded

### DISCUSSION THREAD

**Discussion by sdw4938** [valinet/ExplorerPatcher#960] [General]
![Screenshot 2022-03-01 133425](https://user-images.githubusercontent.com/76626288/156228788-34c824f1-69af-4cff-9505-483b2c45e7e2.png)
Explorer Patcher: 22000.493.42.24
Windows 11 Build 22000.527

**Comment by valinet** [comment]
Quite a few other elements still have not rounded corners. I just finished a sprint working on this. I know about them. I’d concentrate more on ironing out quirks with whatever is already in there, as generally work in this never finishes, I’d be great to get to a release soon.
